lessonbuilder/tool/src/webapp/templates/ShowPage.html

```diff
@@ -2310,18 +2310,31 @@ <h4 class="panel-title">
-                                            <input type="text" rsf:id="question-multiplechoice-answer" name="question-multiplechoice-answer" class="question-multiplechoice-answer" />
-                                            <a role="button" href="#" onclick="deleteAnswer($(this)); return false;" style="display:none" class="deleteAnswerLink"><img src="$context/images/delete.png" rsf:id="question-mc-del"></img></a>
-                                        </div>
-                                        <div id="extraMultipleChoiceAnswers"></div>
-                                    </div>
+                                    <table id="multipleChoiceAnswers" class="table table-striped table-bordered table-hover">
+                                        <thead>
+                                            <tr>
+                                                <th><span rsf:id="msg=simplepage.question-tableheader-selectcorrect">Select Correct Answer(s)</span></th>
+                                                <th><span rsf:id="msg=simplepage.question-tableheader-answertext">Answer Text</span></th>
+                                                <th><span rsf:id="msg=simplepage.question-tableheader-remove">Remove?</span></th>
+                                            </tr>
+                                        </thead>
+                                        <tbody id="multipleChoiceAnswersBody">
+                                            <tr id="copyableMultipleChoiceAnswerDiv">
+                                                <input type="hidden" rsf:id="question-multiplechoice-answer-complete" class="question-multiplechoice-answer-complete" />
+                                                <input type="hidden" rsf:id="question-multiplechoice-answer-id" class="question-multiplechoice-answer-id" />
+                                                <td>
+                                                    <label for="question-multiplechoice-answer-correct"><span rsf:id="msg=simplepage.question-correct-answer" class="offscreen"></span></label>
+                                                    <input type="checkbox" name="question-multiplechoice-answer-correct" rsf:id="question-multiplechoice-answer-correct"  class="question-multiplechoice-answer-correct" />
+                                                </td>
+                                                <td>
+                                                    <input type="text" rsf:id="question-multiplechoice-answer" name="question-multiplechoice-answer" class="question-multiplechoice-answer" />
+                                                </td>
+                                                <td>
+                                                    <a role="button" href="#" onclick="deleteAnswer($(this)); return false;" style="display:none" class="deleteAnswerLink"><img src="$context/images/delete.png" rsf:id="question-mc-del"></img></a>
```

Comment:
img needs an alt tag

Reply:
It's not specified in the markup but it has one, looks like the producer adds one:

![image](https://user-images.githubusercontent.com/8653754/132544076-9376ccf6-146d-419f-9b7e-d2215414ff90.png)

See https://github.com/sakaiproject/sakai/blob/master/lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java#L5255

Reply:
wow!